docs/getting_started/tutorial.md

```diff
@@ -329,30 +357,31 @@ In your `blog_index_page.html` file enter the following content:
 ```
 
 Most of this should be familiar from our previous work with the `home_page.html` template, but we'll explain `get_children` a bit later.
-If you are coming from Django, the `pageurl` tag is similar to Django's `url` tag, but takes a Wagtail Page object as an additional argument.
+If you have a Django background, then you can notice that the `pageurl` tag is similar to Django's `url` tag, but takes a Wagtail Page object as an additional argument.
 
-Now we can create our new page. Here is how we will create one from the Wagtail admin interface: 
+Now you can create a new page. Here is how you can create one from the Wagtail admin interface: 
 
-1.  In the Wagtail admin, go to Pages, then Home.
-2.  Add a child page to the Home page by clicking on the "Actions" icon and selecting the option "Add child page".
-3.  Choose "Blog index page" from the list of the page types.
-4.  Use "Our Blog" as your page title, make sure it has the slug "blog" on the Promote tab, and publish it.
+1. Go to `http://127.0.0.1:8000/admin` and sign in with your admin user details.
+2. In the Wagtail [Admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface), go to Pages, then click Home.
+3. Add a child page to the Home page by clicking **...** at the top of the screen and selecting the option **Add child page**.
+4.  Choose **Blog index page** from the list of the page types.
+5.  Use "Our Blog" as your page title, make sure it has the slug "blog" on the Promote tab, and publish it.
 
-You should now be able to access the url `http://127.0.0.1:8000/blog` on your site (note how the slug from the Promote tab defines the page URL).
+You can now access the url, `http://127.0.0.1:8000/blog` on your site. This gives you an error page showing "TemplateDoesNotExist" because you are yet to create a template for the new page. Also, note how the slug from the Promote tab defines the page URL.
 
-Now we need a model and template for our blog posts. Edit `blog/models.py` to include:
+Now create a model and template for your blog posts. Edit `blog/models.py` to include:
 
 ```python
 from django.db import models
 
 from wagtail.models import Page
 from wagtail.fields import RichTextField
 from wagtail.admin.panels import FieldPanel
+# add this:
```

Comment:
Note. There are a few times you have put 'add this' , maybe it's worth using the advanced line highlighting feature for some of these code snippets. 

It's a bit fiddly but really nice when wanting to call out 'diffs' for progressively adding to an existing file. 

See an example here. 

https://github.com/wagtail/wagtail/blob/401afe7a8170747ad687801b4c661eeac40a98a7/docs/reference/streamfield/blocks.md?plain=1#L546

Note. It has to be inside an rst block.

Reply:
Here's the docs for the Myst parser version of this.
https://myst-parser.readthedocs.io/en/v0.16.1/syntax/syntax.html?highlight=emphasize-lines#parameterizing-directives

Double check but this may mean you do not have to reach for an RST block straight away.

````
```{code-block} python
:lineno-start: 10
:emphasize-lines: 1, 3

a = 2
print('my 1st line')
print(f'my {a}nd line')
```
````

Reply:
```{code-block} python
:lineno-start: 10
:emphasize-lines: 1, 3

a = 2
print('my 1st line')
print(f'my {a}nd line')
```
I already implemented the rst block but I tried this and it works too.

Reply:
I’ve decided to proceed with the "add this"-style comments for now. I like the idea of the highlighting but:

- If it’s RST-only, this seems like a dealbreaker to me
- Even if it works in Markdown, it’s only based on background color, which wouldn’t be enough for us to meet our accessibility goals.

Reply:
@thibaudcolas @lb-  Should I implement this with the Myst parser then? I will remove the RST blocks and use the "add this"-style comments for now. I can always implement either the Myst parser or RST in a new PR once we make a final decision regarding this.

Reply:
Hey. I agree with Thibaud that simpler is probably better here, an add this type comment gets the job done and it's a good call on the accessibility constraint.

We may want to add something to our writing documentation page to recommend this approach.